chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: dnation-kubernetes-monitoring
-version: 1.0.20
+version: 1.0.21
 appVersion: 1.0.20
```

Comment:
Increase also `appVersion`. 
See the [docs](https://helm.sh/docs/topics/charts/#helm) :
- appVersion: The version of the app that this contains (optional). This needn't be SemVer.

Reply:
done